app/src/main/java/io/github/droidkaigi/confsched/activity/ContributorsActivity.java

```diff
@@ -94,9 +93,21 @@ private void renderSavedContributors(Throwable throwable) {
         Crashlytics.logException(throwable);
 
         List<Contributor> contributors = dao.findAll();
-        ContributorsAdapter contributorsAdapter = new ContributorsAdapter(this);
-        contributorsAdapter.addAll(contributors);
-        binding.recyclerView.setAdapter(contributorsAdapter);
+
+        if (contributors.isEmpty()) {
+            Snackbar.make(binding.getRoot(), R.string.contributors_load_error, Snackbar.LENGTH_LONG).show();
```

Comment:
When GitHubAPI error occurred and there are no contributors data in DB, Show snackbar message.